### PR TITLE
Add expand boxes to Updating eZ Platform

### PR DESCRIPTION
**Pull request by DominikaK**
Changes the version-specific steps in the upgrading guide to expand boxes to make the page easier to browse and more readable.

(all maintained versions)

Note: the expand box is added using `??? note "Title"` instead of the usual `!!! note "Title"`. It is rendered as `<details>` plus `<summary>` and is closed by default.

Note 2: Will have to be modified for branches 1.13 and 2.0 to cover new steps that appear in those versions.

### CODE REVIEW

**Comment by glye** [comment]
Cool. I'm probably not qualified for doc review and dunno how to verify that it works as stated, but fwiw +1 for this change anyway.

File: docs/releases/updating_ez_platform.md [hunk 1]
```diff
@@ -132,85 +132,81 @@ Some versions require updates to the database. Look through [the list of databas
...
--    <copyField source="*_fulltext" dest="text" />
--
--    <!--
-       This field is required since Solr 4
-     -->
-     <field name="_version_" type="long" indexed="true" stored="true" multiValued="false" />
-```
+??? note "Solr Bundle 1.4: Index time boosting"
+
+    Solr Bundle v1.4 introduced among other things index time boosting feature, this involves a slight change to the solr scheme that will need to be applied to your config.
```
Comment (MagdalenaZuba) [inline docs/releases/updating_ez_platform.md]: Solr scheme

File: docs/releases/updating_ez_platform.md [hunk 2]
```diff
@@ -132,85 +132,81 @@ Some versions require updates to the database. Look through [the list of databas
...
+
+    To make sure indexing continues to work, apply the following change, restart solr and reindex your content:
```
Comment (MagdalenaZuba) [inline docs/releases/updating_ez_platform.md]: Solr

**Pull request merged**